packages/core/bootstrap/src/lib/cache-warmer/epics.ts

```diff
@@ -158,6 +161,10 @@ export const warmupSubscriber: Epic<AnyAction, AnyAction, any, EpicDependencies>
     }),
     // on a subscribe action being dispatched, spin up a long lived interval if one doesnt exist yet
     mergeMap(([{ payload, key }]) => {
+      payload.childLastSeenById
```

Comment:
I would move this outside of the `mergeMap` into a `tap` function since it causes side-effects